plugins/module_utils/_api/context/api.py

```diff
@@ -0,0 +1,242 @@
+# -*- coding: utf-8 -*-
+# This code is part of the Ansible collection community.docker, but is an independent component.
+# This particular file, and this file only, is based on the Docker SDK for Python (https://github.com/docker/docker-py/)
+#
+# Copyright (c) 2016-2025 Docker, Inc.
```

Comment:
Why is this copyright be docker? is it copied from some upstream sdk?

Reply:
Yep, it's copied from https://github.com/docker/docker-py/